src/com/sap/piper/tools/ToolDescriptor.groovy

```diff
@@ -107,9 +107,12 @@ class ToolDescriptor implements Serializable {
 
     def verifyVersion(script, configuration) {
 
-        def executable = getToolExecutable(script, configuration, false)
-        if (singleVersion) VersionUtils.verifyVersion(script, name, executable, singleVersion, versionOption)
-        if (multipleVersions) VersionUtils.verifyVersion(script, name, executable, multipleVersions, versionOption)
+        if(versionOption) {
```

Comment:
How does this related to the topic of this pull request (remove neo version check). Seems to be a general fix/improvement?

Reply:
The separation of concerns hasn't been applied properly: besides the version check , VersionUtils.verifyVersion ensures the executabilty of the tool. removing the min-must-have-version only means to de-activate the tool execution checker. So the getVersion()  now serves a execution tests, while the (added) verifyVersion checks the version vector only.

Reply:
But the ```versionOption```should be always available, right? It shouln't be possible to instanciate a ToolDescriptor without ```versionOption```?